app/addons/documents/mango/mango.components.js

```diff
@@ -268,16 +250,39 @@ var MangoIndexEditorController = React.createClass({
       return;
     }
 
-    Actions.saveQuery({
+    Actions.saveIndex({
       database: this.state.database,
       queryCode: this.getMangoEditor().getEditorValue()
     });
   }
 });
 
+var ExplainPage = React.createClass({
```

Comment:
Could you create this ExplainPage using the new `class ExplainPage extends React.Components`
The `React.createClass` way of doing things is deprecated